matt-nate/server.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const PORT = process.env.PORT || 3000;
+
+app.use(express.static('./public'));
+
+
+// Done: POST route needs to parse the body passed in with the request. express.urlencoded() attaches "middleware" to do that
+app.post('/articles', express.urlencoded(), (request, response) => {
+  // Done: This route will receive a new article from the form page, new.html, and log that form data to the console. We will wire this up soon to actually write a record to our persistence layer!
+  console.log(request.body);
+  response.status(201).json(request.body);
+});
+
```

Comment:
we also want to have a "catch all" route that will send a 404 status if a get request is made to any route not previously defined in the server.